sequence_processing_pipeline/QCJob.py

```diff
@@ -208,9 +208,11 @@ def _process_sample_sheet(self):
             s = "Assay value '%s' is not recognized." % header['Assay']
             raise PipelineError(s)
 
-        needs_adapter_trimming = (True if
-                                  header['Assay'] == 'Metagenomic'
-                                  else False)
+        meta_types = [Pipeline.METAGENOMIC_PTYPE,
+                      Pipeline.METATRANSCRIPTOMIC_PTYPE]
+
+        needs_adapter_trimming = (
+            True if header['Assay'] in meta_types else False)
```

Comment:
I don't think `meta_types` is an accurate name; perhaps easier to just do:

`needs_adapter_trimming = header['Assay'] in [Pipeline.METAGENOMIC_PTYPE, Pipeline.METATRANSCRIPTOMIC_PTYPE]`